### ISSUE TITLE
[Discussion] UI / Font / Logo - general stuff

### ISSUE THREAD

**Issue by riidom** [josefnpat/vapor#195]
Some basic plan evolved in IRC, summed up to be discussed by all:

a) we need a basic font for "common text" - the needed char coverage needs to be discussed, will there be localisations of the UI texts and/or game descriptions? In that case, we should look for CJK and cyrillic support, just to be on safe side later on.

b) after this, create a special/logo font, that supports A-Z, 0-9, some punctuation (yes, capitals only) tailored to Vapor, but based on the font picked under a) - pretty much like there is a special Löve-Font; some WIP's possibly show up sooner or later, everyone can throw in an attempt (because making fonts is actually fun) - slightly optional though, if it turns out to be too much work, we will just create the letters that are needed and dont make a .otf or .ttf from them

c) colours! So far it looks like we will base on dawnbringers palette *[1], pick some colours from there; this will concern the UI, website(s?) and the logo.

C&C please :)

---

*[1] http://www.pixeljoint.com/forum/forum_posts.asp?TID=16247

**Comment by rm-code** [comment]
+1 for DawnBringers x32 palette!

**Comment by pablomayobre** [comment]
We still need to chose the colors (32 is way too many), 4 or so will be fine we can then use different shades for each one

**Comment by josefnpat** [comment]
I would like to list a few things that made the previous logo so successful:

1) The V looked like a molecule, which is very representative of "steam" or "vapor"
2) The letters were made of pipes, which can contain "vapor". Fantastic pun.
3) The colors were dark and "edgy" much like steam is.

**Comment by Bobbyjoness** [comment]
I think a dark color scheme is good. As most people like dark for some reason. I do lol. Most game clients are dark so there must something to it. And maybe we can have a lot of red? Lol

**Comment by pablomayobre** [comment]
Well light colors are too distractive (and serious) for game clients, so dark rules!

About the pipes, as I said in the IRC I think the letters are kinda weird, I was thinking on a serif font where the serif are actually the pipes

I would put the molecule in the O so that it looks like VAPÖR, I would also add letters to the atoms so that it is easier to identify it as a water molecule, plus the O of the oxygen would complete the word.

About the red, I think red is a nice color for buttons and such, so probably have it as a secondary color will be fine, it could also be used for the Oxygen atom as it is right now :smile:

**Comment by riidom** [comment]
Well, it went slow the last two weeks, today I was browsing googlefonts. Not yet ready to propose a list of suggestions, though.
After that, I toyed around in Inkscape a little. Not exactly an official suggestion, but thought I show something, more to come "soon"(tm) :)

![vaporlogo_01](https://cloud.githubusercontent.com/assets/437109/7221096/adb1485c-e6de-11e4-9ce6-05bc8a9ac4a3.png)

**Comment by josefnpat** [comment]
Oh snap, that looks great! Does it work inverted as well?

**Comment by pablomayobre** [comment]
It looks awesome, not too much Vapor related but... well whatever that is so sweet!

**Comment by riidom** [comment]
Ok, instead of moving forwards, I decided to start from scratch! But I think I am on a better way now, the new version will be more close to the style of the old one. Here is a first draft:

![2nd try of a logo](http://i.imgur.com/ysBGogE.png)

The shading of the joints is still wrong, and there will be an upper bar too that connects to the letters - at least I want to try and see how it looks. Also the height is not quite right atm, the upperline is slightly descending, but that will be fixed when the upper bar comes.

Mostly I'd like to know if I am hurting any typographic feelings here, if yes, please provide an explanation, or draw an overlay. Also of course any other comments are welcome too, for example if I should stick with the techy vector look of the first idea, or rather go for the cuddlish renderstyle I had in the very first 0.x logo.

**Comment by josefnpat** [comment]
I really like this idea as well!

The only thing I can imagine is plumbers complaining about weird bends. (not being 90deg, 45deg, 30deg, 60deg, etc). It makes it feel rather balloon-like, but I'm sure the material render will make it look better.

In the end, it's very playful, and that's also what makes it good.

No typographic feels have been hurts :cat:

**Comment by riidom** [comment]
next step:

![logo v2](http://i.imgur.com/EcLHZ8Z.png)

**Comment by rm-code** [comment]
IMO the big tube distracts from the text too much. I like the rest though :+1:

**Comment by riidom** [comment]
yes, I got an idea how to adress that issue and still keep the tank, hope I will find time to update soon.

**Comment by josefnpat** [comment]
:heart:

**Comment by riidom** [comment]
![logo v3](http://i.imgur.com/n8yue2W.png)

The coloured lights are more of an experiment - but the tank is less distractive now, right?

**Comment by pablomayobre** [comment]
Toó difficult to ser it

**Comment by EntranceJew** [comment]
I can't see much, but it seems like no matter how you place it the tank is going to unbalance the logo.
It might be worth integrating the tank into one of the letters themselves or placing it behind one of them.

**Comment by rm-code** [comment]
I agree with @EntranceJew

**Comment by riidom** [comment]
Holy cow, almost a month passed ?! Well I did these two tests 1-2 weeks ago roundabout.
Trying some contrast thingy here.. let me know if I should elaborate on one of those concepts, or rather try smth new. 

![vaporlogo2b](https://cloud.githubusercontent.com/assets/437109/7944069/d6e72604-0968-11e5-8d0e-ae465cec88ac.png)

![vaporlogo2c](https://cloud.githubusercontent.com/assets/437109/7944077/deddab12-0968-11e5-96f9-4f183b7af33c.png)

**Comment by josefnpat** [comment]
@riidom Nice! What if you increased the red/blue intensity up?

_sorry for the lack of updates; i've had a lot on my plate._

**Comment by riidom** [comment]
![vaporlogo2c](https://cloud.githubusercontent.com/assets/437109/8069191/f8f960b4-0ef8-11e5-8f9c-38dcd4359166.png)
Yup, like this?

**Comment by josefnpat** [comment]
That's nice man!

**Comment by riidom** [comment]
![vaporlogo2c_mapped](https://cloud.githubusercontent.com/assets/437109/8145708/e1d6d638-1212-11e5-83e1-c883faae9d06.png)
![vaporlogo2c_mapped](https://cloud.githubusercontent.com/assets/437109/8145695/3b14f3de-1212-11e5-92de-754c896a44ce.png)
Attempt to map the image to Db16 palette (should have read more carefully, just noticed we talked about Db32 on top of this issue!)
Not sure if I like it. Edit: Made an additional check, the upper one is properly following the Db16 palette.
I think I will redo the render in a way that it allows for easier palettemapping - pretty new to this sort of workflow.

**Comment by riidom** [comment]
![vaporlogo2c2](https://cloud.githubusercontent.com/assets/437109/8241110/a5d803f2-1607-11e5-91ff-c53db420eb28.png)

Backgrounds are overrated (in other words: I got no good idea so far) - just to show how it can look with DB32 palette.

**Comment by josefnpat** [comment]
I can see the dark purple background working well! Can we get a color palette to go with it?

**Comment by pablomayobre** [comment]
Wow I really like that last one! Here I made one of my own... You may take ideas out of it... I wanted to keep it simple 
![vapor](https://cloud.githubusercontent.com/assets/1095969/8323598/0175e06c-1a1d-11e5-82fc-101567955d2f.png)
You can see it is totally inspired by your new design and the previous one... Keep it up! you are almost there :smile:
